setup.py

```diff
@@ -20,6 +20,7 @@
         'cryptography>=1.7.2',
         'pyOpenSSL>=16.2.0',
         'msgpack-python>=0.4.2',
+        'filemagic'
```

Comment:
I was able to stand-up a Coordinator on Windows 10 after installing a 'file for Windows' port, switching to python-magic, and allowing config for specifying path to magic.mgc